oracle/constants/currency_pairs.go

```diff
@@ -0,0 +1,132 @@
+package constants
```

Comment:
Per Nikhil's point, we should be able to define cannonical representations of the feeds without enforcing any additional criteria (min provider count, decimals, etc). Those can be determined adhoc instead of having to explicitly define that when we might want (dYdX decimals, initia decimals)